ironfish/jest.setup.env.js

```diff
@@ -4,28 +4,6 @@
 const consola = require('consola')
 const { initializeSapling } = require('@ironfish/rust-nodejs')
 
-jest.mock('node-datachannel', () => {
```

Comment:
Just kinda curious what the reason for removing was? Assuming its because the new version of `node-datachannel` doesn't match this shape but then does it need to be mocked everywhere still?

Reply:
Ok nvm I think I understand now because we are passing it down from the SDK

Reply:
Main reason is actually you can't do module-level mocks in `jest` of ES Module packages. I think we could use something like `esmock`, but didn't chase it down too far since I found the current solution.